### ISSUE TITLE
Successfully installed PyVimeo-1.0.11 -- BUT

### ISSUE THREAD

**Issue by mcgeo52** [vimeo/vimeo.php#216]
TLDR:
"Successfully installed PyVimeo-1.0.11 pycurl-7.43.0 six-1.10.0 tinydb-3.13.0 tuspy-0.2.1" doesn't work.

When I try using PyVimeo, I get an error:
ImportError: pycurl: libcurl link-time ssl backend (openssl) is different from compile-time ssl backend (none/other)

***If the install was successful, why am I getting an error?***

Attempts to fix the link-time error were not successful.

Details follow.

Please do not waste my time or yours suggesting anything that I have already tried, as documented here.

Thank you for your help.

  -- Geo. McCalip

DETAILS:

I ran:
sudo pip install PyVimeo

I got:
Successfully installed PyVimeo-1.0.11 pycurl-7.43.0 six-1.10.0 tinydb-3.13.0 tuspy-0.2.1

But then when I ran:
python video-upload-master.py proj003/filev1.csv renders/proj003

I got:
Traceback (most recent call last):
  File "video-upload-master.py", line 6, in <module>
    import vimeo
  File "/usr/local/lib/python2.7/site-packages/vimeo/__init__.py", line 8, in <module>
    from .client import VimeoClient
  File "/usr/local/lib/python2.7/site-packages/vimeo/client.py", line 11, in <module>
    from .upload import UploadMixin
  File "/usr/local/lib/python2.7/site-packages/vimeo/upload.py", line 10, in <module>
    from tusclient import client
  File "/usr/local/lib/python2.7/site-packages/tusclient/client.py", line 1, in <module>
    from tusclient.uploader import Uploader
  File "/usr/local/lib/python2.7/site-packages/tusclient/uploader.py", line 12, in <module>
    from tusclient.request import TusRequest
  File "/usr/local/lib/python2.7/site-packages/tusclient/request.py", line 1, in <module>
    import pycurl
ImportError: pycurl: libcurl link-time ssl backend (openssl) is different from compile-time ssl backend (none/other)

***If the install was successful, why am I getting an error?***

After researching the lib curl link-time error, I tried:
sudo pip uninstall pycurl

I got:
Successfully uninstalled pycurl-7.43.0

I then tried:
export PYCURL_SSL_LIBRARY=nss
sudo pip install --compile --install-option="--with-nss" --no-cache-dir pycurl 

I got:
Successfully installed pycurl-7.43.0.3

Then when I ran:
python video-upload-master.py proj003/filev1.csv renders/proj003

I still got:
Traceback (most recent call last):
  File "video-upload-master.py", line 6, in <module>
    import vimeo
  File "/usr/local/lib/python2.7/site-packages/vimeo/__init__.py", line 8, in <module>
    from .client import VimeoClient
  File "/usr/local/lib/python2.7/site-packages/vimeo/client.py", line 11, in <module>
    from .upload import UploadMixin
  File "/usr/local/lib/python2.7/site-packages/vimeo/upload.py", line 10, in <module>
    from tusclient import client
  File "/usr/local/lib/python2.7/site-packages/tusclient/client.py", line 1, in <module>
    from tusclient.uploader import Uploader
  File "/usr/local/lib/python2.7/site-packages/tusclient/uploader.py", line 12, in <module>
    from tusclient.request import TusRequest
  File "/usr/local/lib/python2.7/site-packages/tusclient/request.py", line 1, in <module>
    import pycurl
ImportError: pycurl: libcurl link-time ssl backend (openssl) is different from compile-time ssl backend (none/other)

So I ran:
sudo pip uninstall PyVimeo

And got:
Successfully uninstalled PyVimeo-1.0.11

And then I ran:
sudo pip install PyVimeo

And got:
Successfully installed PyVimeo-1.0.11

Note: No mention of pycurl-7.43.0 six-1.10.0 tinydb-3.13.0 tuspy-0.2.1

But, since I had "Successfully installed PyVimeo-1.0.11" I tried:
python video-upload-master.py proj003/filev1.csv renders/proj003

I still got:
Traceback (most recent call last):
  File "video-upload-master.py", line 6, in <module>
    import vimeo
  File "/usr/local/lib/python2.7/site-packages/vimeo/__init__.py", line 8, in <module>
    from .client import VimeoClient
  File "/usr/local/lib/python2.7/site-packages/vimeo/client.py", line 11, in <module>
    from .upload import UploadMixin
  File "/usr/local/lib/python2.7/site-packages/vimeo/upload.py", line 10, in <module>
    from tusclient import client
  File "/usr/local/lib/python2.7/site-packages/tusclient/client.py", line 1, in <module>
    from tusclient.uploader import Uploader
  File "/usr/local/lib/python2.7/site-packages/tusclient/uploader.py", line 12, in <module>
    from tusclient.request import TusRequest
  File "/usr/local/lib/python2.7/site-packages/tusclient/request.py", line 1, in <module>
    import pycurl
ImportError: pycurl: libcurl link-time ssl backend (openssl) is different from compile-time ssl backend (none/other)

**Comment by tommypenner** [comment]
Wrong library.

https://github.com/vimeo/vimeo.py/issues
